azure-pipelines-v3.yml

```diff
@@ -9,6 +9,7 @@ pr:
 
 variables:
 - group: devresourcekeyvault
+- group: DocfxCIKVPub
 - name: dotnetVersion
```

Comment:
We should use another KV in a specific tenant. Please check my comment under the feature.

Reply:
@herohua KV moved to PME

Reply:
@herohua KV moved to PME